pkgs/top-level/all-packages.nix

```diff
@@ -22021,7 +22021,10 @@ with pkgs;
 
   mtpfs = callPackage ../tools/filesystems/mtpfs { };
 
-  mtxclient = callPackage ../development/libraries/mtxclient { };
+  mtxclient = callPackage ../development/libraries/mtxclient {
+    # https://github.com/NixOS/nixpkgs/issues/201254
+    stdenv = if stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU then gcc11Stdenv else stdenv;
```

Comment:
Is there a specific reason for gcc*11*Stdenv?

Reply:
Any new enough gcc should work, but as we are using gcc 11 on all other platforms, i think it's a safe option to stick with it.